sqldb/sqlc/migrations/000002_amp_invoices.up.sql

```diff
@@ -1,5 +1,5 @@
--- amp_invoices_payments  
-CREATE TABLE IF NOT EXISTS amp_invoice_payments (
+-- amp_invoices 
+CREATE TABLE IF NOT EXISTS amp_invoices (
```

Comment:
IUUC, this is still actually an AMP payment? 

Since a regular invoice and an AMP invoice aren't really all that different. What's different is that an AMP invoice can have several payments to it, each with distinct HTLCs and `set_id` values.

Reply:
FWIW, much of the fields in the existing `invoice` table don't necessarily apply here. Eg: there's no global `preimage` for an AMP invoice.